srcpkgs/rofi-rbw/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'rofi-rbw'
+pkgname=rofi-rbw
+version=0.5.0
+revision=1
+build_style=python3-pep517
+make_install_target="rofi_rbw-${version}-py3-none-any.whl"
+hostmakedepends="python3-setuptools python3-wheel"
+depends="python3 python3-ConfigArgParse rbw"
```

Comment:
needs `xdotool` (xorg) or `wtype` (wayland) to work properly

Reply:
That's true, but should I put both as dependencies? I don't use x11, so xdotool is a pointless dependency for me.

Reply:
no